test/Amadevus.RecordGenerator.TestsBase/ValidatingRecord.cs

```diff
@@ -14,9 +14,9 @@ public partial class ValidatingRecord
         /// </summary>
         public string Switch { get; }
 
-        static partial void Validate(ref string name, ref string @switch)
+        partial void Validate()
         {
-            if (name is null) throw new ArgumentNullException(nameof(name));
+            if (Name is null) throw new ArgumentNullException("name");
```

Comment:
I'd keep this as `nameof(Name)`. Different casing is not a huge issue here IMO, but more importantly any refactoring won't "break" this.